README.md

```diff
@@ -26,21 +26,21 @@ from armorblox import client
 c = client.Client(api_key='your-api-key-here', instance_name='yourtenantname')
 
 # Fetch a list of threats
-threat_incidents = c.threats.list()
+threat_incidents, next_page_token, total_incident_count = c.threats.list()
 
 # Fetch a specific threat
 incident = c.threats.get(44006)
 
 
 # Fetch a list of abuse incidents
-abuse_incidents = c.abuse_incidents.list()
+abuse_incidents, next_page_token, total_incident_count = c.abuse_incidents.list()
 
 # Fetch a specific abuse incident
 abuse_incident = c.abuse_incidents.get(44200)
 
 
 # Fetch a list of DLP incidents
-dlp_incidents = c.dlp_incidents.list()
+dlp_incidents, next_page_token, total_incident_count = c.dlp_incidents.list()
 
```

Comment:
Thanks for updating the README! Can you also add another example here with `c.incidents.list()` which shows how to use the page token in a loop to fetch all incidents?

Reply:
Sure, added it now.